samcli/local/common/runtime_template.py

```diff
@@ -83,11 +83,13 @@ def get_local_lambda_images_location(mapping, runtime):
     return os.path.join(_lambda_images_templates, runtime, dir_name + "-lambda-image")
 
 
-SUPPORTED_DEP_MANAGERS: Set[str] = {
-    c["dependency_manager"]  # type: ignore
-    for c in list(itertools.chain(*(RUNTIME_DEP_TEMPLATE_MAPPING.values())))
-    if c["dependency_manager"]
-}
+SUPPORTED_DEP_MANAGERS: List[str] = list(
```

Comment:
This and `LAMBDA_IMAGES_RUNTIMES` was changed to list due to a mypy error: `samcli/commands/init/command.py:195: error: Argument 1 to "Choice" has incompatible type "Set[Any]"; expected "Sequence[str]"`.

Instead of just a list, I could do `list(set()`, to ensure only one value is present.